### ISSUE TITLE
Provide guide on connecting to ATmega32U4

### ISSUE THREAD

**Issue by heri16** [nerves-project/nerves_system_linkit#1]
How do we interface with the ATmega32U4? Is there a library to control the ATmega32U4 in real time from elixir?

**Comment by fhunleth** [comment]
The ATmega is connected to the MIPS's UART0 (ttyS0). You can use `nerves_uart` to communicate with it.

With any luck, the docs and functionality of the LinkIt Smart Nerves integration will be updated at Nerves Camp on July 17th.

**Comment by mobileoverlord** [comment]
There is an example project in nerves-examples that illustrates communicating with the ATmega32U4 using the firmata firmware to blink. @fhunleth do we want to document this on the main readme?

**Comment by fhunleth** [comment]
Yeah, this does seem like an important thing to document. Thanks for revisiting this issue. I had dropped the ball on this one with all the prep for Nerves Camp.

**Comment by heri16** [comment]
Which is the best firmata driver on elixir? Is there a MIDI library that
can be built upon?
